SECURITY.md

```diff
@@ -0,0 +1,78 @@
+# Helm Security Process and Policy
+
+This document provides the details on the Helm security policy and details the processes
+surrounding security handling including a how to guide on reporting a security vulnerability
+for anything within the Helm organization.
+
+## Report A Vulnerability
+
+We’re extremely grateful for security researchers and users that report vulnerabilities
+to the Helm community. All reports are thoroughly investigated by a set of Helm maintainers.
+
+To make a report please email the private security list at cncf-helm-security@lists.cncf.io
+with the details.
+
+You may, but are not required to, encrypt your email to this list using the PGP keys
+of security team members, listed below.
+
+Name               | Key URL                                              | Fingerprint
+------------------ | ---------------------------------------------------- | -----------
+Adam Reese         | [link](https://keybase.io/adamreese/pgp_keys.asc)    | 49D0 9C86 C3DC 8DA3 F0A0 7622 1EF6 1234 7F8A 9958
+Adnan Abdulhussein | [link](https://keybase.io/prydonius/pgp_keys.asc)    | 5111 DA73 DF12 D8E8 12CA 462F 2CDB BFBB 37AE 822A
+Justin Scott       ||
+Matt Butcher       | [link](https://keybase.io/technosophos/pgp_keys.asc) | ABA2 5295 98F6 626C 420D 335B 62F4 9E74 7D91 1B60
+Matt Farina        | [link](https://keybase.io/mattfarina/pgp_keys.asc)   | 672C 657B E06B 4B30 969C 4A57 4614 49C2 5E36 B98E
+Matt Fisher        | [link](https://keybase.io/bacongobbler/pgp_keys.asc) | 967F 8AC5 E221 6F9F 4FD2 70AD 92AA 783C BAAE 8E3B
+Michelle Noorali   ||
+Nikhil Manchanda   | [link](https://keybase.io/slicknik/pgp_keys.asc)     | 5674 7211 71F8 CD59 F285 353D 9692 757E BCD3 D66C
+Vic Iglesias       | [link](https://keybase.io/vicnastea/pgp_keys.asc)    | 43A7 994E F6C7 7D7F 147B C152 381B ABCF F47D EAFC
+
+### When To Send A Report
+
+You think you have found a vulnerability in a Helm project or a dependency of a Helm project. This can be any of the repositories on the [Helm GitHub organization](https://github.com/helm).
+
+### When Not To Send A Report
+
+* If a vulnerability has been found in an application deployed by a Helm Chart. Instead, contact the application maintainers
+* For guidance on securing Helm. Please see the documentation or ask in one of the support channels
```

Comment:
nit: comma instead of period after `on securing Helm` ?

Reply:
done, done, and done